src/librustdoc/html/render.rs

```diff
@@ -1387,7 +1403,7 @@ impl Context {
 
         let mut themes = self.shared.themes.clone();
         let sidebar = "<p class='location'>Settings</p><div class='sidebar-elems'></div>";
-        themes.push(PathBuf::from("settings.css"));
+        themes.push(ThemePath { path: PathBuf::from("settings.css"), disabled: false });
```

Comment:
I just realized that it's weird to have `settings.css` into `themes`. Maybe rename `themes` into something else like `styleFiles`?